### DISCUSSION TITLE
Dotnet tool should not be relied on for wixsharp.

### DISCUSSION THREAD

**Discussion by smaudet** [oleg-shilo/wixsharp#1582] [General]
Basically, dotnet should not be relied upon for this toolset. It is un-necessary and a poor design decision.

There is no guarantee that dotnet will be present on a particular system, especially while targeting older frameworks.

Possibly in future there may be redirections/additional restrictions in place on the usage (of that executable), the name may change, making using the library more difficult/error prone, etc.

@oleg-shilo If you like I can create a new bug/issue for a feature request for this.

**Comment by smaudet** [comment]
WRT to https://github.com/oleg-shilo/wixsharp/issues/1546, I will just say, I was not upset, nor attacking you at all. There is something called, "dry humor". I don't agree with you, and you were really not reading my replies either. I wonder why you think I was being heated in one thread when I've not done so on any other of my replies?

If you are getting emotional advice from a robot, I can see where you maybe struggle. :) By the way, why the double standard? Did you apply your robot to any of your replies?

If you like, you can unlock that thread so that the bug can actually be useful and perhaps delete your replies. I can also remove mine and we could continue our discussion here.

**Reply by oleg-shilo** [reply]
@smaudet, disguising confrontational attitude as humor is a very common reason for conversations to go sour. Particularly when people disagree with each other.

The reason for testing your (and mine too) comments with ChatGPT was that I did not want you to try to dismiss my view as emotion based. And it looks like the robot has a good grasp of a respectful human conversation.

Anyway, your last paragraph above indicates your still prefer to continue the discussion #1546 and you also give me your opinion on what you call "your maybe struggle". This is despite the fact that you logged a few issues yesterday, during which you maintained a completely respectful conversation style. 

I prefer us to focus only at the problem at hand. So, I am locking this conversation, too.

Nevertheless, I am going to process this and other issues you logged with the full attention they deserve.

**Reply by oleg-shilo** [reply]
I have created an issue for this topic and it will be handled there: #1585.
